_topic_maps/_topic_map.yml

```diff
@@ -176,7 +176,7 @@ Topics:
     File: installing-aws-china
   - Name: Installing a cluster on AWS using CloudFormation templates
     File: installing-aws-user-infra
-  - Name: Installing a cluster using AWS Local Zones
+  - Name: Installing a cluster by using AWS Local Zones
```

Comment:
I changed this because of an IBM Style Guide requirement:

![Screenshot from 2023-08-04 09-38-50](https://github.com/openshift/openshift-docs/assets/57954076/48fe439c-c913-49f1-bb37-132e92c0319b)

I changed the title from `Installing a cluster by using AWS Local Zones` to `Installing a cluster on AWS with worker nodes on AWS Local Zones`.